examples/node/cli.rs

```diff
@@ -941,57 +958,90 @@ fn list_offers(node: &LightningNode) -> Result<()> {
-
-        println!("{kind} offer created at {created_at}:");
-        println!("      Expires at:         {expires_at:?}");
-        println!(
-            "      Amount:             {}",
-            amount_to_string(offer.amount)
-        );
-        println!("      LNURL-w:            {:?}", offer.lnurlw);
-        match offer.offer_kind {
-            OfferKind::Pocket {
-                id,
-                exchange_rate,
-                topup_value_minor_units,
-                exchange_fee_minor_units,
-                exchange_fee_rate_permyriad,
-                error,
-                ..
-            } => {
-                println!("                   ID:    {id}");
-                println!(
-                    "      Value exchanged:    {:.2} {}",
-                    topup_value_minor_units as f64 / 100f64,
-                    exchange_rate.currency_code,
-                );
-                println!(
-                    "      Exchange fee rate:  {}%",
-                    exchange_fee_rate_permyriad as f64 / 100_f64
-                );
-                println!(
-                    "      Exchange fee value: {:.2} {}",
-                    exchange_fee_minor_units as f64 / 100f64,
-                    exchange_rate.currency_code,
-                );
-                let exchanged_at: DateTime<Utc> = exchange_rate.updated_at.into();
+    Ok(())
+}
+
+fn print_offer(offer: &OfferInfo) {
+    let kind = match offer.offer_kind {
+        OfferKind::Pocket { .. } => "Pocket",
+    };
+
+    let created_at: DateTime<Local> = offer.created_at.into();
+    let expires_at: Option<DateTime<Local>> = offer.expires_at.map(|e| e.into());
+
+    println!("{kind} offer created at {created_at}:");
+    println!("      Expires at:         {expires_at:?}");
+    println!(
+        "      Amount:             {}",
+        amount_to_string(&offer.amount)
+    );
+    println!("      LNURL-w:            {:?}", offer.lnurlw);
+    match &offer.offer_kind {
+        OfferKind::Pocket {
+            id,
+            exchange_rate,
+            topup_value_minor_units,
+            exchange_fee_minor_units,
+            exchange_fee_rate_permyriad,
+            error,
+            ..
+        } => {
+            println!("                   ID:    {id}");
+            println!(
+                "      Value exchanged:    {:.2} {}",
+                *topup_value_minor_units as f64 / 100f64,
+                exchange_rate.currency_code,
+            );
+            println!(
+                "      Exchange fee rate:  {}%",
+                *exchange_fee_rate_permyriad as f64 / 100_f64
+            );
+            println!(
+                "      Exchange fee value: {:.2} {}",
+                *exchange_fee_minor_units as f64 / 100f64,
+                exchange_rate.currency_code,
+            );
+            let exchanged_at: DateTime<Utc> = exchange_rate.updated_at.into();
+            println!(
+                "             Exchange at: {}",
```

Comment:
Probably `exchanged at:`, but who cares. This is only the CLI tool.

Reply:
Fixed anyway :) [1d9d904](https://github.com/getlipa/lipa-lightning-lib/pull/917/commits/1d9d9045e7db641cd7c1e0c091e582f7394d7136)